parsers/CL.py

```diff
@@ -141,37 +145,49 @@ def production_processor_historical(raw_data):
     return ordered_data
 
 
-@refetch_frequency(timedelta(days=1))
-def fetch_production(
-    zone_key: str = "CL-SEN",
-    session: Session | None = None,
-    target_datetime: datetime | None = None,
+def _fetch_production(
+    zone_key: ZoneKey,
+    session: Session | None,
+    target_datetime: datetime | None,
     logger: Logger = getLogger(__name__),
-):
-    if target_datetime is None and ENABLE_LIVE_PARSER:
+    enable_live_parser: bool = ENABLE_LIVE_PARSER,
```

Comment:
And why is this passed as a argument if it's only assigned the constant?

It would be simpler to use the constant directly as before.

Reply:
Yes, parameter `enable_live_parser` in the `_fetch_production` function has a default value of `ENABLE_LIVE_PARSER`. This way the `fetch_production` function will produce the same results as before, as live parser is disabled.
However, it is now possible to call the `_fetch_production` function with parameter `enable_live_parser` set to `True` from a unit test, in order to cover the code processing live data. This is in fact done in the newly added test `test_snapshot_live_data`.